docs/concepts/storage/volumes.md

```diff
@@ -320,6 +321,19 @@ simultaneous writers allowed.
 
 See the [iSCSI example](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/volumes/iscsi) for more details.
 
+### fc(fibre channel)
+
+A `fc` volume allows an existing fibre channel volume to be mounted into your pod.
```

Comment:
The rule is to us "an" for words that sound like they start with a vowel even if the first letter is a consonant. In this case, since 'fc' is an acronym, so it would be pronounced EFF-CEE. Consequently, because the leading sound is a vowel, "an" should be used.